### PR TITLE
feat(rpc): support rpc runs on service worker

### PR DESCRIPTION
**Pull request by IronLu233**
# Description

This PR add support of running lumos RPC on Webworker.

In previous, lumos RPC use axios as http requester. But it is not capable on Webworker. In this PR, when detect code runs on Webworker, it will use fetch as axios adapter.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Website
- [ ] Example
- [ ] Other

# How Has This Been Tested?

- [x] Add a unit test for modifying axios adapter.
- [x] Manually test on [Nexus](https://github.com/ckb-js/nexus)

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by homura** [comment]
@IronLu233 prettier check is failed, can you please help to run `npx lint-staged` and commit it agian?

**Comment by homura** [comment]
The `@vespaiach/axios-fetch-adapter` [does'n work well](https://github.com/ckb-js/lumos/actions/runs/4340202413/jobs/7578523793#step:7:104) with ava.js, maybe we can create an `axios-fetch-adapter.js` and `axios-fetch-adapter.d.ts` as a temporary solution

File: website/yarn.lock [hunk 1]
```diff
@@ -1343,6 +1343,125 @@
   resolved "https://registry.yarnpkg.com/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz#75a2e8b51cb758a7553d6804a5932d7aace75c39"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
+"@ckb-lumos/base@^0.19.0":
```
Comment (homura) [inline website/yarn.lock]: why lumos@0.19 is installed?

Comment (IronLu233) [reply website/yarn.lock]: I'm not sure, I never change the limits. But reverting it is necessary

**Review by homura — APPROVED** [review]

**Pull request merged**